test/d2l-custom-teacher-course-creation-confirm.test.js

```diff
@@ -83,7 +83,7 @@ describe('d2l-teacher-course-creation-confirm', () => {
 
 		it('finish button with error triggers change-page event', async() => {
 			const errorMessage = 'Houston we have a problem';
-			const patches = { createCourse: async() => {throw new Error(errorMessage);}};
+			const patches = { createCourse: async() => {throw new Error(errorMessage);} };
```

Comment:
if you're going to fix one, might as well fix the other
```suggestion
			const patches = { createCourse: async() => { throw new Error(errorMessage);} };
```